MsixCore/msixmgr/msixmgr.cpp

```diff
@@ -801,7 +870,15 @@ int main(int argc, char * argv[])
                     std::wcout << "Finished unpacking packages to: " << unpackDestination << std::endl;
                     std::wcout << std::endl;
 
-                    OutputUnpackFailures(packageSourcePath, skippedFiles, failedPackages, failedPackagesErrors);
+                    if (outputFilePath.empty())

```

Comment:
>  if (outputFilePath.empty()) [](http://example.com/codeflow?start=18&length=29)

Not sure it makes sense to have this be an either\or case. As long as so much other logging is still going to the command line, it seems like this stuff should still go there too even if output path is set. If outputpath is set it can be duplicated to the file.

Reply:
If output path is set this would end up being logged in the file